src/test/java/org/influxdb/BatchOptionsTest.java

```diff
@@ -130,7 +132,8 @@ public void testFlushDuration() throws InterruptedException {
       
       //check no points writen to DB before the flush duration
       QueryResult result = influxDB.query(new Query("select * from weather", dbName));
-      Assertions.assertNull(result.getResults().get(0).getSeries());
+      List<Series> series = result.getResults().get(0).getSeries();
```

Comment:
Any reason why a test must be changed ?

Reply:
Because:
   +in JSON response (for an empty query), the 'series' element does not exists
   + in MessagePack response  (for an empty query), series exist as an empty array
--> I think none of them are not wrong, just the consistency from server side. I am actually thinking to open an issue for the server